website/pages/en/tutorials.js

```diff
@@ -290,6 +290,22 @@ class Tutorials extends React.Component {
                   "tutorials/start-a-private-network/"
                 )}`
               },
+              {
+                img: `${baseUrl}img/ink-smart-contracts-tutorial.png`,
+                title: <translate>ink! Smart Contracts Tutorial</translate>,
+                text: (
+                  <translate>
+                    A comprehensive, end-to-end tutorial for building an ERC20
+                    token contract using ink!.
+                  </translate>
+                ),
+                difficulty: "easy",
+                length: "4",
+                prerequisite: false,
+                version: "2.0.0-dev",
```

Comment:
I don't think we should introduce a new, ad hoc version.

Reply:
To this end, did you actually go through this tutorial step-by-step to ensure that it works with `rc3` and the screenshots, etc, are at least mostly up-to-date?

Reply:
I don't see any problem with this new tag. The problem is that many times this tutorial moves at different speed.
I tried at the rc3 tag but it doesn't works due to a small bug which is fixed in master. THis is why I use `dev` instead of `rc3`

Reply:
I don't want to enable a tutorial if it doesn't work on a released version.

Reply:
This is the exact problem I had in recipes that made me start experimenting with gitflow. The classic situation is:

There is a bug in Substrate's latest release (currently rc3)  that prevents a devhub product from working (e.g. contracts workshop). The bug is fixed on master and the devhub product works when using master. But because we want to stick to releases only, we cannot publish until another release is cut.

Don't have a concrete solution, but it's worth noticing this is not an isolated case. This will also affect other teams building on Substrate who might say, "Our chain has a bug because Substrate has a bug. We'd like to release a new node that contains the fix, but Substrate itself hasn't released yet, so we're stuck."

Reply:
I think this is a good time to go by the "three versions back" rule. If there is a released version within the last three versions, we can keep the tutorial up...otherwise, it doesn't seem like the feature is stable/important enough to maintain a tutorial for it.

Reply:
A problem with teh last three versions is what I had to do in the past 2-3 months for this tutorial until it had many patches. It was using alpha.6 because it worked and also the UI. THe alpha.6 needed an older nightly to build (1st patch). The second problem came when the UI followed more breaking changes from master and I added another patch to tell the users about the data types. At the end the tutorial was pretty broken because you had to patch rustc and the UI.

Reply:
seems like it would be worth considering gitflow for devhub as this isn't an isolated case-- a lot of feedback loops dependent on bug patches --

Reply:
> I think this is a good time to go by the "three versions back" rule. If there is a released version within the last three versions, we can keep the tutorial up...otherwise, it doesn't seem like the feature is stable/important enough to maintain a tutorial for it.

yeap this-- maybe we should consider a /b/ side to devhub or sandbox where we can play with tutorials that require more experimental, less stable, builders in mind-- or people who are advanced enough they are willing to debug and patch broken rust/UI